en/metadata.yaml

```diff
@@ -36,6 +36,13 @@ navigation:
       - title: Introduction
         location: stacks/index.md
 
+  - title: Manage a device
```

Comment:
I'm not sure about the Build a Stack topics, but I do think this topic belongs under Manage a Device. Do you have a suggestion? I'd leave it here.